docs/pages/web/utilities/scrollboundarycontainer.js

```diff
@@ -567,7 +103,7 @@ function ScrollBoundaryContainerExample() {
       **[Modal](/web/modal)** / **[OverlayPanel](/web/overlaypanel)**
       Modal and OverlayPanel come with ScrollBoundaryContainer built-in, so any anchored components used in their children tree should work out-of-the-box. Passing an additional ScrollBoundaryContainer will break the existing styling on scroll.
 
-      **[Tooltip](/web/tooltip)** / **[Popover](/web/popover)** / **[Dropdown](/web/dropdown)** / **[ComboBox](/web/combobox)**
+      **[Tooltip](/web/tooltip)** / **[Popover](/web/popover)** / **[Dropdown](/web/dropdown)** / **[ComboBox](/web/combobox) / **[HelperButton](/web/helperButton)** / **[PopoverEducational](/web/popovereducational)** **
```

Comment:
Let's not add these links until they're functional, i.e. the components exist

(also note that it's HelpButton, not HelperButton)

Reply:
DONE